reuelcj/christa-reuel-ux-aop
# Local storage API

Create a data structure using `localStorage` to hold user data for the app to use

## What It Does

- [ ] The localStorage API module meets the requirements described in the [localStorage API specification](https://github.com/Banno/ux-intern-onboarding/blob/master/resources/specs/localStorage-api-spec.md), including the following functions:
  - [x] `createUser`
  - [x] `getUser`
  - [x] `getUsers`
  - [x] `updateUser`
  - [x] `deleteUser`
  - [x] `generateId`
  - [x] `validateUser`
  - [ ] any optional functions or helpers
- [ ] All API code is located in `src/api/`

## To Test
Follow instructions at the bottom of 'src/api/toTestInConsole.js' and check the localstorage in your browser to verify.
create-user.html is where I pulled in the localStorage API . 

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:
